usaspending_api/common/filters/time_period/decorators/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .new_awards_only import NewAwardsOnlyTimePeriod
```

Comment:
> And can time_period be used in a common way, across all django "apps" (i.e. domain areas / bounded contexts ... like awards, accounts, etc.)?

It can be used across all django apps. Anything that uses a time period filter, which include multiple apps.

Reply:
I can move it to filters though.

Reply:
Ok, i see it deals with dates generically. Still, may want to do some restructuring of the concrete impls to organize by their implied domain areas. I can take this to an offline conversation.

Reply:
That'd be great, like maybe put the abstract class in common, then put app specific implementations in their app. So transaction search implementation would go in the search app?

Reply:
Let me know if you'd like me to do so in this PR. More than happy to do it.

Reply:
Ideas I'm thinking

option 1:
- move this abstract class up into `__init__.py` as it's truly the base class of a series of concrete classes that may fall into this `time_period` package.
- then within that package, have domain-specific modules that capture the implementation. E.g. in cases where this base class is being used against awards and transactions, create a `...time_period.awards.py` module in the `time_period` package. That module can have both the `NewAwardsOnlyTimePeriod` and `TransactionSearchTimePeriod` classes in it (it's best to store related classes in a module in python, like you would store them in a folder/package in java or .net)

option 2:
- remove the `time_period` package
- have a `...filters.time_period.py` module that has the base Abstract class declared
- move all of the domain-specific impls into e.g. `usaspending_api.awards.v2.filters` 

I tend to like option 1 better if it the impls don't get too entangled with HTTP API-specific features like requests and responses. If the smell like HTTP API handling, move them down to the option 2 level.

Reply:
Ok will do.